initialize.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
```

Comment:
I generally add `set -o errexit` at the beginning of my scripts. This makes it so if one command fails, the execution stops. Add if you don't want to do subsequent steps if a previous step failed.